### ISSUE TITLE
Unknown option character `X' in: -Xlinker

### ISSUE THREAD

**Issue by asotog** [CocoaPods/CocoaPods#3334]
Have a swift project using pods version 0.36.1  (initially only objective-c pods dependencies, building fine),
Podfile = 
```
source 'https://github.com/CocoaPods/Specs.git'
platform :ios, "8.1"


pod 'MBProgressHUD', :git => 'https://github.com/jmnavarro/MBProgressHUD', :branch => 'cancel-delayed-hide'
pod 'SMXMLDocument'
pod 'UICKeyChainStore'
pod 'DTPickerPresenter'
pod 'TNRadioButtonGroup'
pod 'MDRadialProgress'
pod 'ODRefreshControl'
pod 'Liferay-iOS-SDK', '6.2.0.11'
pod 'DateTools'

```

so i needed to add a swift pod dependency and got "Unknown option character `X' in: -Xlinker." on project build, dependency podfile line looks like = 
```
pod 'Swift-Validator', :git => 'https://github.com/asotog/swift-validator'
```

reading issues history found that ``use_frameworks!`` is required on pod file when using swift dependencies (so added that to pods file and now linker errors gone) but now the rests of libraries began to complain, for example in bridging header, multiple errors like  "file not found" (#import "MBProgressHUD.h" ... ..) 

In summary, have a swift project that needs objective-c and swift libraries/pods, thats how can be reproduced the issue

Thanks in advance, hope this ticket works for you

**Comment by kylef** [comment]
In Objective-C, you shouldn't be importing the files with quotes, but instead like the following:

```objective-c
#import < MBProgressHUD/MBProgressHUD.h>
```

When using pods from Swift with `use_frameworks!`, you shouldn't put them in the bridging file, but instead import them in your swift sources that depend on them. Like so:

```swift
import MBProgressHUD
```

**Comment by asotog** [comment]
importing from swift files works fine but we have other custom objective-c classes that needs to import from pods objective-c dependencies, thats where the issue happens, actually i cant even import as #import < ... > its throwing that module does not exists

**Comment by segiddins** [comment]
In Objective-C, when integrating as a framework, you should be using `@import Framework;`
